src/oas.md

```diff
@@ -2556,13 +2556,13 @@ The runtime expression is defined by the following [ABNF](https://tools.ietf.org
                     ; %x2F ('/') and %x7E ('~') are excluded from 'unescaped'
     escaped         = "~" ( "0" / "1" )
                     ; representing '~' and '/', respectively
-    name = *( CHAR )
+    name = *char
```

Comment:
Note: RFC7159 and RFC8259 both name this rule `char`. ABNF rule names are case-insensitive, but it is still easier to check the reference if the casing is exactly the same.